src/pages/Filing/FilingApp/FilingOverviewPage.tsx

```diff
@@ -34,48 +37,58 @@ export default function FilingOverview(): ReactElement {
     isLoading: associatedInstitutionsLoading,
   } = useAssociatedInstitutions();
 
-  if (associatedInstitutionsLoading) return <LoadingContent />;
+  const {
+    data: filingPeriods,
+    error: filingPeriodsError,
+    isLoading: filingPeriodsLoading,
+  } = useFilingPeriods();
+
+  if (associatedInstitutionsLoading || filingPeriodsLoading)
+    return <LoadingContent />;
+
+  // TODO: Implement logic to derive current filing period based on current date
+  // https://github.com/cfpb/sbl-frontend/issues/546
```

Comment:
@shindigira The filing periods endpoint includes start/end/due dates for each filing period, so it may not be as simple as just "the current year"...or it might be, we'll need to check with the Regs folks. 

On HMDA they also have Quarterly filing periods, which I don't think applies to SBL but is another item to verify with the Regs folks.

```
[
  {
    "code": "string",
    "description": "string",
    "start_period": "2024-05-21T20:49:34.045Z",
    "end_period": "2024-05-21T20:49:34.045Z",
    "due": "2024-05-21T20:49:34.045Z",
    "filing_type": "ANNUAL"
  }
]
```